src/modules/GamesAPI.js

```diff
@@ -3,11 +3,16 @@ import API_KEY from '../../config.js';
 const GAMES_API_BASE_URL = 'https://mmo-games.p.rapidapi.com';
 const HOST = 'mmo-games.p.rapidapi.com';
 
+// export let static pageNumber = 0;
+
```

Comment:
I think you forgot this one :smiley:. You can remove this debugging comment when merging

Reply:
oops! yeah, I forgot.